test-d/browser.test-d.tsx

```diff
@@ -12,8 +12,6 @@ Honeybadger.configure({
   revision: 'git SHA/project version',
   component: 'example_comonent',
   action: 'example_action',
-  onerror: true,
-  onunhandledrejection: true,
```

Comment:
Noticed these aren't used anywhere in the code so I removed them. Relics from a previous version, maybe?

Reply:
Ah yeah, these got renamed to `enableUncaught` and `enableUnhandledRejection`.